### DISCUSSION TITLE
show image in table

### DISCUSSION THREAD

**Discussion by hasanmonfared** [orchidsoftware/platform#1585] [Q&A]
hi, I stored a photo with Storage class  in folder and save url in  the database and I am reading it in database and showing the link on the table, but I want to show it in the img tag.
I looked in the documents but I did not find anything. Thank you for your help.

**Comment by tabuna** [comment]
Hi, you can handle the cell content yourself, for example:

```php
TD::make('image')
    ->render(function (Repository $repository) {
        return "<img src='https://...";
    });
```

Also, you can use Blade templates:

```php
TD::make('image')
    ->render(function (Repository $repository) {
        return view('your.template');
    });
```

You can also use Blade components:

```php
TD::make('image')->component(YourBladeComponent::class, 'nameOfVariable');
```

**Reply by hasanmonfared** [reply]
hi, Hello, thanks for your answer, When I use the above code, I get the following error.

`Argument 1 passed to App\Orchid\Layouts\Services\ServicesListLayout::App\Orchid\Layouts\Services\{closure}() must be an instance of Orchid\Screen\Repository, instance of App\Models\Services given, called in /var/www/html/afiyatbashe-main/vendor/laravel/framework/src/Illuminate/Support/helpers.php on line 383 (View: /var/www/html/afiyatbashe-main/resources/views/vendor/platform/layouts/table.blade.php)`

I do not know how to place the data and put  **image url** in **src**
Thank you for your help

**Reply by tabuna** [reply]
Hi, the code I provided was very general. In your case, the model comes to the closure function.

```php
use App\Models\Services;


TD::make('image')
    ->render(function (Services $services) {
        return "<img src='https://...";
    });
```
